<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="false">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>784</width>
    <height>559</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>784</width>
    <height>559</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pixArk Server Manager</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>761</width>
     <height>551</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>759</width>
      <height>549</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Server path and status</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>731</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="location_text"/>
       </item>
       <item>
        <widget class="QPushButton" name="location_button">
         <property name="text">
          <string>Set Location</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>172</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Server status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status">
         <property name="text">
          <string>Not Running</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>261</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="update_server">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_server">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_server">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTabWidget" name="Admin">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>761</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Administration</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>731</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Server Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="server_name"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>731</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Server Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="server_pass"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Admin Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="admin_pass"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>731</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="port"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Cube Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cube_port"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Query Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="q_port"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Rcon Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rcon_port"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Players:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="players_no"/>
        </item>
        <item>
         <widget class="QCheckBox" name="battle_eye">
          <property name="text">
           <string>Disable BattleEye</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>World Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="world_name"/>
        </item>
       </layout>
      </widget>
      <widget class="QWebEngineView" name="webEngineView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>139</y>
         <width>731</width>
         <height>231</height>
        </rect>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Server Options</string>
      </attribute>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>301</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Under Construction</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
